packages/amplify-util-mock/src/api/api.ts

```diff
@@ -383,12 +386,18 @@ export class APITest {
     if (!name) {
       throw new AmplifyFault('MockProcessFault', {
         message: 'No AppSync API is added to the project',
-        link: AMPLIFY_SUPPORT_DOCS.CLI_GRAPHQL_TROUBLESHOOTING.url,
+        resolution: `Use 'amplify init' in the root of your app directory to create a new environment.`,
```

Comment:
If the reason is `No AppSync API is added to the project`, then resolution should be to `Add an AppSync API using 'amplify add api' in your project`?

Reply:
Shouldn't this be an `AmplifyError` instead of `AmplifyFault`

Reply:
ya Phani, updated the resolution! thank you!

@Amplifiyer , true technically both AmplifyError & AmplifyFault can be used here to throw an exception. But if you look here in the `amplify-exception.ts` file where the type of error `MockProcessFault` is of type `AmplifyFaultType`. for reference: https://github.com/aws-amplify/amplify-cli/blob/8e44b0da1af532a90a8e39c880f7d7a0db197e81/packages/amplify-cli-core/src/errors/amplify-exception.ts#L200